### ISSUE TITLE
Pip install wagtail-modeltranslation 0.8a1 not working

### ISSUE THREAD

**Issue by agelinas** [infoportugal/wagtail-modeltranslation#165]
with **pip **install wagtail-modeltranslaton****, I get a list requirment already satisfied and the new version of wagtail modeltranltaion does not install;

with **pip install **wagtail-modeltranslation==0.8a1****,  I get:

Collecting wagtail-modeltranslation==0.8a1
Using cached wagtail-modeltranslation-0.8a1.tar.gz
Requirement already satisfied: wagtail>=1.4 in /home/alain/CanHTF/lib/python3.5/site-packages (from wagtail-modeltranslation==0.8a1)
Collecting django-modeltranslation>0.12.1 (from wagtail-modeltranslation==0.8a1)
 Could not find a version that satisfies the requirement django-modeltranslation> ...

I will wait a more recent packaging, I guess,

Thank you,
I'm using version 0.6.0rc2 and it's doing well.

**Comment by timstallmann** [comment]
You can specify a particular git tag in your `requirements.txt` file, like this:
`-e git+https://github.com/infoportugal/wagtail-modeltranslation.git@v0.8a1#egg=wagtail_modeltranslation`

**Comment by dmarcelino** [comment]
The issue is likely because the alpha setup has `dependency-links`. Try:

```
pip install --process-dependency-links wagtail-modeltranslation==0.8a1
```

We probably should document that somewhere.

**Comment by timstallmann** [comment]
FWIW I get the following error when running `pip install --process-dependency-links wagtail-modeltranslation==0.8a1`

```
Collecting django-modeltranslation>0.12.1 (from wagtail-modeltranslation==0.8a1)
  Could not find a version that satisfies the requirement django-modeltranslation>0.12.1 (from wagtail-modeltranslation==0.8a1) (from versions: 0.2, 0.3, 0.3.1, 0.3.2, 0.3.3, 0.4.0a1, 0.4.0b1, 0.4.0b2, 0.4.1, 0.5a1, 0.5b1, 0.5, 0.5.1, 0.6, 0.6.1, 0.7, 0.7.1, 0.7.2, 0.7.3, 0.8b1, 0.8b2, 0.8, 0.8.1, 0.9, 0.9.1, 0.10, 0.10.1, 0.10.2, 0.11rc1, 0.11rc2, 0.11, 0.12, 0.12.1)
No matching distribution found for django-modeltranslation>0.12.1 (from wagtail-modeltranslation==0.8a1)
```

**Comment by dmarcelino** [comment]
@timstallmann , I just tried running `pip install --process-dependency-links wagtail-modeltranslation==0.8a1` on a fresh virtualenv and it installed django-modeltranslation from github fine. What version of `pip` are you using? Mine is `9.0.1`.

**Comment by dmarcelino** [comment]
Closing this since `v0.8` has been released (#168) and the dependency-links are gone.
